__tests__/app.test.js

```diff
@@ -145,4 +146,31 @@ describe("GET /api/users", () => {
             })
         })
     });
+});
+
+describe("GET /api/reviews", () => {
+    test('200: responds with a reviews array containing the correct properties', () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.reviews).toHaveLength(13);
+            response.body.reviews.forEach((review) => {
+                console.log(review)
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        owner: expect.any(String),
+                        title: expect.any(String),
+                        review_id: expect.any(Number),
+                        category: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number)
+                    })
+                )
```

Comment:
Hey Javier!

You should have written another test to implement the additional comment count behaviour. When following TDD practice we do not want to change our previous, passing tests as we want them to still pass even with some extra functionality.

Please could you revert your original 200 test for 'GET api/reviews' and then add another test to handle the additional comment_count behaviour